internal/desktop/assets/app.css

```
@@ -0,0 +1,1115 @@
+	color: #e6b4a8;
+	font-size: 9px;
+	text-align: right;
+}
+.sr-only {
+	position: absolute;
+	width: 1px;
+	height: 1px;
+	padding: 0;
+	margin: -1px;
+	overflow: hidden;
+	clip: rect(0, 0, 0, 0);
```

Stylelint (17.14.0)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)